app/models/concerns/aliquotable.rb

```diff
@@ -50,19 +50,50 @@ def available_volume
       primary_aliquot.volume - used_volume
     end
 
-    # Method to check if there is enough volume available.
-    # It compares the available volume with the required volume.
+    # Method: available_volume_check
     #
-    # @param required_volume [Numeric] The volume required for the operation.
+    # This method is used to validate if the available volume is sufficient for a required volume.
     #
-    # @return [Boolean] Returns true if the available volume is greater than
-    # or equal to the required volume.
-    # If not, it adds an error message to `errors` and returns false.
-    def volume_check(required_volume)
+    # The method performs the following checks:
+    # 1. If the available volume is greater than or equal to the required volume,
+    # the method returns true.
+    # 2. If the available volume is less than the required volume, the method adds an error to the
+    # base attribute of the record and returns false.
+    #
+    # @param required_volume [Numeric] The volume that is required.
+    #
+    # @return [Boolean] Returns true if the available volume is sufficient, false otherwise.
+    def available_volume_check(required_volume)
```

Comment:
Hi @seenanair, just suggesting here; does a method name like `is_available_volume_sufficient` convey the purpose better? Same with the `used_volume_check`.

Reply:
Yes, that would be more descriptive 👍 . I will change it to available_volume_sufficient (As RuboCop suggested to Rename `is_available_volume_sufficient` to `available_volume_sufficient?` ) and primary_aliquot_volume_sufficient (as we are checking it's sufficiency rather than used_volume)